pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -166,7 +172,18 @@ func (m *Manager) Start() error {
 		for {
 			select {
 			case isShuttingDown := <-events:
-				klog.V(1).InfoS("Shutdown manager detected new shutdown event, isNodeShuttingDownNow", "event", isShuttingDown)
+				var shutdownType string
+				if isShuttingDown {
+					shutdownType = "shutdown"
+				} else {
+					shutdownType = "cancelled"
+				}
+				klog.V(1).InfoS("Shutdown manager detected new shutdown event", "event", shutdownType)
+				if isShuttingDown {
+					m.recorder.Eventf(m.nodeRef, v1.EventTypeNormal, kubeletevents.NodeShutdown, "Shutdown manager detected new shutdown event at %v", time.Now().Format(time.RFC3339))
```

Comment:
do we need to explicitly add the time to the event message? 

Events should already have `eventTime` field on the event itself so I'm not sure if we need to add a time in the message itself: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#event-v1-core

Reply:
Fixed. Thanks!